push_processor/handler.py

```diff
@@ -18,68 +18,66 @@
 here_dir = os.path.abspath(os.path.dirname(__file__))
```

Comment:
you only use this once, in an __init__ for the class. You could just move it there and get rid of the last global.